utils/bip70-utils.js

```diff
@@ -303,6 +303,18 @@ const signAndPublishPaymentRequestTransaction = async (
   let totalUtxoAmount = 0;
   const transactionBuilder = new SLP.TransactionBuilder("mainnet");
 
+  let opReturnLength = null;
+  if (
+    paymentRequest.outputs[0].amount.eq(0) &&
+    !paymentRequest.outputs[0].tokenAmount
+  ) {
+    const asBuffer = Buffer.from(paymentRequest.outputs[0].script, "hex");
+    opReturnLength = asBuffer.length;
```

Comment:
Are you able to use byteLength here? It should be more accurate

Reply:
Updated this to use `Buffer.byeLength()`